checkpointers/empty/checkpointer.go

```diff
@@ -0,0 +1,34 @@
+package emptycheckpointer
+
+import (
+	k "github.com/remind101/kinesumer/interface"
```

Comment:
You can probably get rid of this and just use `https://github.com/aws/aws-sdk-go/blob/master/service/kinesis/kinesisiface/interface.go` instead.